akka-docs/src/test/java/jdocs/dispatcher/MyUnboundedMessageQueueSemantics.java

```diff
@@ -4,8 +4,8 @@
 
 package jdocs.dispatcher;
 
-//#mailbox-implementation-example
+//#mailbox-marker-interface
 // Marker interface used for mailbox requirements mapping
 public interface MyUnboundedMessageQueueSemantics {
 }
-//#mailbox-implementation-example
+//#mailbox-marker-interface
```

Comment:
Refer to [this comment](https://github.com/akka/akka/pull/23191/files#r122614439).